game_eggs/steamcmd_servers/hurtworld/egg-hurtworld.json

```diff
@@ -1,30 +1,30 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-01-20T12:55:06-05:00",
+    "exported_at": "2023-10-07T18:12:51+08:00",
     "name": "Hurtworld",
     "author": "brycea@terrahost.cloud",
     "description": "Hurtworld is a hardcore multiplayer survival FPS with a focus on deep survival progression that doesn't become trivial once you establish some basic needs. Built for hardcore gamers, Hurtworld aims to punish.",
     "features": [
         "steam_disk_space"
     ],
-    "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
-    ],
+    "docker_images": {
+        "quay.io\/parkervcp\/pterodactyl-images:debian_source": "quay.io\/parkervcp\/pterodactyl-images:debian_source"
```

Comment:
Please change the image to:
`ghcr.io/parkervcp/steamcmd:debian`

The image you use is fucking old and not longer supported

Reply:
> Please change the image to: `ghcr.io/parkervcp/steamcmd:debian`
> 
> The quay image you use is fucking old and not longer supported
Changed